module-gui/gui/widgets/text/Text.cpp

```diff
@@ -186,6 +186,22 @@ namespace gui
         drawLines();
     }
 
+    void Text::removeFromCursor()
+    {
+        const auto currentText       = getText();
+        const auto currentTextLength = currentText.length();
+        const auto cursorPosition    = cursor->getAbsolutePosition();
+        const auto substrLength      = currentTextLength - cursorPosition;
```

Comment:
[minor] No need to introduce a distinct variable in line 192 for this

Reply:
fixed